baseframe/forms/validators.py

```diff
@@ -397,6 +398,21 @@ def __call__(self, form, field):
 # Legacy name
 ValidEmailDomain = ValidEmail
 
+class IsNotHttps(object):
+    """
+    Validator to confirm that a URL has 'https://'
+    """
+
+    default_message = __(u'The URL provided is not HTTPS')
+
+    def __init__(self, message=None, message_urltext=None):
+        self.message = message or self.default_message
+
+    def __call__(self, form, field):
+        if field.data:
+            if "https://" not in field.data:
```

Comment:
This will pass if the URL has `https://` in the middle. You want `field.data.startswith`.